development/test-log-collector/cmd/app/mainerr.go

```diff
@@ -27,15 +27,14 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	restclient "k8s.io/client-go/rest"
 
-	"github.com/kyma-project/test-infra/development/test-log-collector/pkg/hyperscaler"
-
 	"github.com/kyma-project/test-infra/development/test-log-collector/pkg/resources/clustertestsuite"
 	octopusTypes "github.com/kyma-project/test-infra/development/test-log-collector/pkg/resources/clustertestsuite/types"
 )
 
 type config struct {
 	SlackToken     string
 	ConfigLocation string
+	ProwJobType    string `envconfig:"default=not-specified"` // post-master-kyma-gke-integration etc
```

Comment:
Variable name should be rather ProwJobName. ProwJob types are presubmit, postsubmit or periodic.